api/v1beta2/ibmvpccluster_types.go

```diff
@@ -98,26 +111,108 @@ type AdditionalListenerSpec struct {
 	Port int64 `json:"port"`
 }
 
-// VPCNetworkSpec defines the desired state of the network resources for the cluster for extended VPC Infrastructure support.
-type VPCNetworkSpec struct {
-	// workerSubnets is a set of Subnet's which define the Worker subnets.
+// BackendPoolSpec defines the desired configuration of a VPC Load Balancer Backend Pool.
+type BackendPoolSpec struct {
+	// name defines the name of the Backend Pool.
 	// +optional
-	WorkerSubnets []Subnet `json:"workerSubnets,omitempty"`
+	Name *string `json:"name,omitempty"`
+
+	// algorithm defines the load balancing algorithm to use.
+	// +required
+	Algorithm string `json:"algorithm"`
```

Comment:
lets add a validation here for the `Allowable values: [least_connections,round_robin,weighted_round_robin]` and also a default value set when no option mentioned